pages/about.html

```diff
@@ -1,12 +1,42 @@
+<!-- should include some information about you, your interests, background or 
+    similar. Only post what you are comfortable sharing. -->
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../styles/style.css" rel="stylesheet">
+    <link rel="icon" type="image/png" sizes="16x16" href="../images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Zen+Loop&display=swap" rel="stylesheet">
+    <title>Morgan Adkisson</title>
 </head>
-<body>
-  
+<body class="wrapper">
+    <aside class="side-containers"></aside>
+    <header class="nav-bar">
+        <div class="width-container">
+            <h1 class="name">Morgan <span>Adkisson</span></h1>
```

Comment:
Since these classes only occur once in each HTML document, it is ok to make them IDs rather than classes!